gradle.properties

```diff
@@ -23,4 +23,4 @@ android.enableJetifier=true
 kotlin.code.style=official
 # Needed to enable Android data binding.
 android.databinding.enableV2=true
-
+android.jetifier.blacklist=bcprov-jdk15on
```

Comment:
This needs a comment, I think, to explain what it does & why it's necessary.

Reply:
I have added the comment